elisp-bytecode.texi

```diff
@@ -896,6 +935,16 @@ Call @code{list} with three arguments.
 @kindex list4
 Call @code{list} with four arguments.
 
+@node listN
+@unnumberedsubsubsec @code{list4} (70)
+@kindex listN
+Call @code{list} an arbitrary number of arguments.
```

Comment:
Again, "up to 255".

Reply:
Thanks. Have fixed the xxxN, and will push when I am finished with the comments.